typescript-primitives-and-variables/main.ts

```diff
@@ -0,0 +1,18 @@
+const fullName: string = 'Katelyn Ngo';
+const isCool: boolean = true;
+const totalPets: number = 1;
+
+console.log(' Value of Full Name:', fullName);
+console.log('Value of isCool:', isCool);
+console.log('Value of totalPets:', totalPets);
+console.log('typeof fullName:', typeof fullName);
+console.log('type of isCool:', typeof true);
+console.log('typeof totalPets:', typeof 1);
```

Comment:
Same point here, you should be using the typeof totalPets so we are using the variable value and not a number.